bee-hive/bee-hive.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Bee-hive</title>
+</head>
+<style>
+    body {color: black; background-color: lightblue;}
+</style>
+<body>
+    <h1>
+        Bee-hive
+    </h1>
+    <form action="/" method="post" enctype="multipart/form-data">
+        <h2>Select a agent defintion file and workflow difinition file</h2>
+        <div style="display: flex;">
+            <input type="file" name="agents"  placeholder="Agent definition" style="color: blue;">
+            <input type="file" name="workflow"  placeholder="Workflow definition" style="color: blue;">
+        </div>
```

Comment:
let's do a separate `<div>` for agent file and workflow file, just to make it harder for users to accidentally use the wrong box for the different yaml files

Reply:
maybe something like

```html
        <h2>Select an agent definition file</h2>
        <div style="display: flex;">
            <input type="file" name="agents"  placeholder="Agent definition" style="color: blue;">
        </div>
        <h2>Select a workflow definition file</h2>
        <div style="display: flex;">
            <input type="file" name="workflow"  placeholder="Workflow definition" style="color: blue;">
        </div>
```

Reply:
yes, it looks better.  Thanks!